### PR TITLE
[INJIWEB-511]: Fixed Radio Button Toast

### PR DESCRIPTION
**Pull request by kamlesh012**

### CODE REVIEW

File: inji-web/src/components/DataShare/DataShareContent.tsx [hunk 1]
```diff
@@ -14,6 +15,9 @@ export const DataShareContent:React.FC<DSContentProps> = (props) => {
     const dispatch = useDispatch();
     const {t} = useTranslation("DataShareExpiryModal");
 
+    const handleClick = () => {
```
Comment (vijay151096) [inline inji-web/src/components/DataShare/DataShareContent.tsx:18]: make it inline

Comment (kamlesh012) [reply inji-web/src/components/DataShare/DataShareContent.tsx]: done.

File: inji-web/src/components/DataShare/DataShareContent.tsx [hunk 2]
```diff
@@ -58,15 +58,22 @@ export const DataShareContent:React.FC<DSContentProps> = (props) => {
...
+                    <div className="relative inline-block">
+                        <input 
+                            type="radio" 
+                            name="consentValidity" 
+                            value="date" 
+                            disabled={true} 
+                            className="accent-iw-primary scale-150" 
+                        />
+                        <div 
+                            className="absolute inset-0"
```
Comment (vijay151096) [inline inji-web/src/components/DataShare/DataShareContent.tsx:71]: why is this div needed ?

Comment (kamlesh012) [reply inji-web/src/components/DataShare/DataShareContent.tsx:71]: to make the disabled radio button react to events.  without wrapper div disabled radio button is not responding to events.

**Review by vijay151096 — APPROVED** [review]
lgtm

**Pull request merged**